.travis.yml

```diff
@@ -58,6 +58,14 @@ jobs:
           skip_cleanup: true
           on:
             tags: true
+    - stage: Publish latest Docker Image on Daily Build
```

Comment:
How can ci control it is run once a day?

Reply:
`condition: $TRAVIS_EVENT_TYPE = cron` it is based on this condition
currently cron job is configured to run daily once